sys/include/net/ng_fib.h

```diff
@@ -34,6 +34,14 @@ typedef struct rrp_address_msg_t {
     uint32_t address_flags; /**< The flags for the given address */
 } rrp_address_msg_t;
 
+/**
+ * @brief entry used to collect available destinations
+ */
+typedef struct fib_destination_set_entry_t {
+    uint8_t dest[16];    /**< The destination address */
```

Comment:
we could wait for #3022 to set the size of `dest[]` to `UNIVERSAL_ADDRESS_SIZE` instead of this magic number.
I can also make  a PR to change this later on, since RPL is using ipv6 addresses of 16 bytes anyways